packages/@okta/vuepress-theme-prose/assets/css/components/_top-bar.scss

```diff
@@ -282,12 +282,11 @@
               position: absolute;
               background-color: white;
 
-              a {
-                color: cv('cerise');
+              a,div {
+                text-decoration: none;
                 width: 100%;
                 padding: 1.0rem 1.3rem;
```

Comment:
Not totally sure if this is the right line, but would prefer less padding between items in the dropdown menus (matching the marketing pages).